praw/models/reddit/subreddit.py

```diff
@@ -1722,8 +1738,59 @@ def comments(self):
            for comment in reddit.subreddit('iama').stream.comments():
                print(comment)
 
+        Using a ``pause_after`` limit to stop querying for comments after
+        60 failed data fetches can be done by:
+
+        .. code:: python
+
+           rdev = reddit.subreddit('redditdev')
+           for com in rdev.stream.comments(pause_after=60):
+               if com is None:
+                   break
+               print(com)
+
+        To resume fetching comments after hitting a ``pause_after limit``,
+        try:
+
+        .. code:: python
+
+           help = reddit.subreddit('help')
+           help_stream = help.stream.comments(pause_after=100)
```

Comment:
Consider using smaller values. With 1 minute delays between the latter network requests that result in no new data, a `pause_after` value of 100 will likely take over an hour before timing out. I imagine people will likely want to use this feature for approximately a 1-5 minute (pause_after of 4-10) timeout in addition to the no-delay timeout.